### ISSUE TITLE
Cout is ambigous

### ISSUE THREAD

**Issue by codecreed20** [microsoft/vscode#184261]
Type: <b>Bug</b>

c++ says cout is ambigous it worked fine recently 

VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-11400H @ 2.70GHz (12 x 2688)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.77GB (7.54GB free)|
|Process Argv|D:\\DSA C++\\bitwiseop.cpp --crash-reporter-id 7f3fcf16-9a1b-42ae-a340-05cc48c37255|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
code-runner|for|0.12.0
cmake-tools|ms-|1.14.31
cpptools|ms-|1.16.0
cpptools-extension-pack|ms-|1.3.0
cmake|twx|0.0.17

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30730052
hidesbindicator:30730055
pythongtdpath:30739704
pythonnosmt12:30757131

```

</details>

**Comment by sean-mcmanus** [comment]
@codecreed20 This was from the cpptools extension -- it's fixed with the latest 1.16.3 release.

@aeschli This was an important regression caused by the 1.16.0 (pre-release) of our cpptools extension. If any C++ issues from users with the cpptools extension installed could be moved to https://github.com/Microsoft/vscode-cpptools/issues that would be great.

**Comment by isidorn** [comment]
@sean-mcmanus thanks for letting us know. I am sorry that this issue fell through the cracks. We usually move issues to the cpp tools repo. But @aeschli is on a 2 month vacation so it was unfortunate timing.
